app/index.html

```diff
@@ -14,6 +14,21 @@
                 box-sizing: border-box;
             }
 
+            ::-webkit-scrollbar {
+                width: var(--dui-width-scrollbar);
+                height: var(--dui-width-scrollbar);
+                background-color: var(--dui-color-background-scrollbar);
+            }
+
+            ::-webkit-scrollbar-track {
+                background-color: var(--dui-color-background-scrollbar);
+            }
+
+            ::-webkit-scrollbar-thumb {
+                border-radius: var(--dui-radius-scrollbar-border);
+                background-color: var(--dui-color-foreground-scrollbar);
+            }
+
```

Comment:
We may not need to add the prefix webkit.
Can you verify? The autoprefixer should automatically prefix.

Reply:
These are not standard css variables and only available on webkit browsers